.github/workflows/src/summarize-checks.js

```diff
@@ -0,0 +1,88 @@
+// import { setEquals } from "../../shared/src/equality.js";
+import { extractInputs } from "./context.js";
+// import { PER_PAGE_MAX } from "./github.js";
+
+/**
+ * @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments
+ * @returns {Promise<void>}
+ */
+export default async function summarizeChecks({ github, context, core }) {
+  let owner = process.env.OWNER || "";
+  let repo = process.env.REPO || "";
+  let issue_number = parseInt(process.env.ISSUE_NUMBER || "");
+  let head_sha = process.env.HEAD_SHA || "";
+
+  if (!owner || !repo || !issue_number || !head_sha) {
+    let inputs = await extractInputs(github, context, core);
+    owner = owner || inputs.owner;
+    repo = repo || inputs.repo;
+    issue_number = issue_number || inputs.issue_number;
+    head_sha = head_sha || inputs.head_sha;
+  }
```

Comment:
```suggestion
  const { owner, repo, head_sha, issue_number } = await extractInputs(
    github,
    context,
    core,
  );
```

I think this should be sufficient.  Copied from here:

https://github.com/Azure/azure-rest-api-specs/blob/a13f38e7991f2135a9d4eef3533a4aa4f83897ea/.github/workflows/src/set-status.js#L26